src/wxProfilerGUI/profilergui.cpp

```diff
@@ -62,6 +62,7 @@ static const wxCmdLineEntryDesc g_cmdLineDesc[] =
 	{ wxCMD_LINE_SWITCH, "h", "", "Displays help on the command line parameters.",			wxCMD_LINE_VAL_NONE, wxCMD_LINE_OPTION_HELP },
 	{ wxCMD_LINE_OPTION, "r", "", "Runs an executable and profiles it.",					wxCMD_LINE_VAL_STRING, wxCMD_LINE_PARAM_OPTIONAL|wxCMD_LINE_NEEDS_SEPARATOR },
 	{ wxCMD_LINE_OPTION, "a", "", "Attaches to a process (by its PID) and profiles it.",	wxCMD_LINE_VAL_STRING, wxCMD_LINE_PARAM_OPTIONAL|wxCMD_LINE_NEEDS_SEPARATOR },
+	{ wxCMD_LINE_OPTION, "thread", "", "Profiles the specified thread(s) in the process, multiple threads must be in a comma-delimited list (See -a for specifying the process ID).",	wxCMD_LINE_VAL_STRING, wxCMD_LINE_PARAM_OPTIONAL|wxCMD_LINE_NEEDS_SEPARATOR },

```

Comment:
Should this perhaps specify how to specify the threads (decimal thread ID?)?

Reply:
I wanted to be able to specify multiple threads. I couldn't find anything baked-in to wxWidgets's command line framework to allow multiple values. So, it's a string that we manually split and parse.

Reply:
Right, but that's not what I mean. I mean that perhaps the help text for the switch should explain what exactly is the expected format and contents of those comma-delimited values.

Reply:
None of the other args have examples. Perhaps the format of this arg is unusual.
I don't mind doing this.